src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1413,9 +1401,7 @@ public function dumpParameter($name)
 
     /**
      * @deprecated since version 2.6.2, to be removed in 3.0.
-     *             Use \Symfony\Component\DependencyInjection\ContainerBuilder::addExpressionLanguageProvider instead.
-     *
-     * @param ExpressionFunctionProviderInterface $provider
+     * Use \Symfony\Component\DependencyInjection\ContainerBuilder::addExpressionLanguageProvider instead.
```

Comment:
I think the previous version is more readable and as it renders the same, I would not change this.

Reply:
This is something I can't cover easily. I recommend reverting those manually.